server/selection_test.go

```diff
@@ -496,7 +497,13 @@ func TestMinLSSelector_SelectUnknownSession_RandFreq(t *testing.T) {
 	sel.Add(sessions)
 
 	// When randFreq = 1.0 we should select randomly instead of selecting the session with the most stake
-	sess := sel.selectUnknownSession(context.TODO())
+	var sess *BroadcastSession
+	for i := 0; i < 10; i++ {
+		sess = sel.selectUnknownSession(context.TODO())
+		if !bytes.Equal(sess.OrchestratorInfo.TicketParams.Recipient, addr.Bytes()) {
```

Comment:
f16de03ba49817a7f49b9313a7f52ef8bb1e0a4c swaps `bytes.Equal` in for the `cmp.Equal` call using `go-cmp`.